backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -386,6 +393,16 @@ private Bundle convertRowToFhirBundle(TestResultRow row, UUID orgId) {
           StringUtils.isEmpty(testOrderedCode)
               ? MultiplexUtils.inferMultiplexTestOrderLoinc(deviceTypeDiseaseEntries)
               : testOrderedCode;
+
+      String finalTestOrderedCode = testOrderedCode;
+      testOrderedLoincLongName =
+          deviceTypeDiseaseEntries.stream()
+              .filter(
+                  disease ->
+                      Objects.equals(disease.getTestOrderedLoincCode(), finalTestOrderedCode))
```

Comment:
could we use either an abbreviation like `dtd` or the full `deviceTypeDisease` here instead of `disease` to make clear we're not iterating through a list of diseases? Think it's relatively clear from context here, but was momentarily confused as to why we were trying to get a test ordered loinc from a disease object.